### ISSUE TITLE
tabnet mask loss bug

### ISSUE THREAD

**Issue by kaiyu-tang** [titu1994/tf-TabNet#13]
in tabnet.,py line 292  entropy_loss = 0.
this will conduct the entropy_loss always 0

**Comment by atamazian** [comment]
It seems to be fixed; requesting this issue to be closed.
